darwin.m

```diff
@@ -0,0 +1,67 @@
+// +build darwin
+
+#import <Foundation/NSArray.h>
```

Comment:
@fffw Have you looked at elevate/cocoasudo?  I really think we can just get rid of this code and use that.